nano/core_test/active_transactions.cpp

```diff
@@ -967,6 +967,89 @@ TEST (active_transactions, confirmation_consistency)
 }
 }
 
+/**
+ * This test case tests the creation of reverse links for incoming confirmed blocks.
+ */
+TEST (active_transactions, reverse_link)
+{
+	// Initialise node environment and enable reverse links
+	nano::system system;
+	nano::node_config node_config = nano::node_config (nano::get_available_port (), system.logging);
+	node_config.enable_reverse_links = true;
+	auto & node = *system.add_node (node_config);
+
+	// Generate a random key pair
```

Comment:
Are you aware that you can use something like:
```
	wallet.insert_adhoc (nano::dev::genesis_key.prv);
	auto const send1 = wallet.send_action (nano::dev::genesis_key.pub, key1.pub, node.config.receive_minimum.number ());
```
to do transaction pairs (send/recv) easily?

Reply:
> Are you aware that you can use something like:
> 
> ```
> 	wallet.insert_adhoc (nano::dev::genesis_key.prv);
> 	auto const send1 = wallet.send_action (nano::dev::genesis_key.pub, key1.pub, node.config.receive_minimum.number ());
> ```
> 
> to do transaction pairs (send/recv) easily?

Oh, this looks indeed cleaner if I can drop the vote stuff then. I will give it a try. Thanks 😄

Reply:
Just remember that autoreceive only works for receivables larger or equal to: `node.config.receive_minimum.number ()`

Reply:
> Just remember that wallet auto-receive only works for receivables larger or equal to: `node.config.receive_minimum.number ()`

Thanks, `send_action` and `receive_action` worked great. 😊